src/main/java/hello/HelloController.java

```diff
@@ -4,10 +4,23 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @RestController
-public class HelloController {
-    
+public class HelloController implements Comparable {
+    public static String foo = "";
+
     @RequestMapping("/")
     public String index() {
         return "Greetings from Spring Boot!";
     }
-}
+
+    public Object clone() {
+        return null;
+    }
+
+    public int compareTo(Object value) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube:9000/coding_rules#rule_key=squid%3AS1210)